test/webdriverio/test/move_test.ts

```diff
@@ -218,3 +277,22 @@ function getConnectedBlockInfo(browser: Browser, id: string, index: number) {
     index,
   );
 }
+
+/**
+ * Given a block ID, get the coordinates of that block, as returned by
+ * .getRelativeTosSurfaceXY().
```

Comment:
```suggestion
 * getRelativeToSurfaceXY().
```

Nit: typo fix, plus removed the '.' since it doesn't seem necessary for the code reference.